src/current/cockroachcloud/costs.md

```diff
@@ -189,25 +215,37 @@ Customers will not be charged for data transfer for CockroachDB {{ site.data.pro
 
 For CockroachDB Standard and Advanced, CockroachDB Cloud bills directly to customers based on their usage with a zero-dollar rate during the preview period.
 
-The usage data is based on the volume of data moved (per GiB) and the cloud provider’s line item for each type of data transfer, which CockroachDB categorizes as **Same-region data transfer**, **Cross-Region data transfer**, or **Public Internet data transfer**.
+The usage data is based on the volume of data moved (per GiB) and the cloud provider’s identified data transfer type, which CockroachDB categorizes as **Same-region data transfer**, **Cross-Region data transfer**, or **Public Internet data transfer**. Each category is reflected as a line item on your invoice.
```

Comment:
"usage data"? Do youy mean "Usage is based on..."? Maybe "the type of data transfer, as identified by the Cloud provider. Data transfer is classified as ...."